src/include/duckdb/main/query_profiler.hpp

```diff
@@ -155,7 +157,11 @@ class QueryProfiler {
 	//! console)
 	string save_location;
 
-private:
+	idx_t OperatorSize() {
+		return tree_map.size();
+	}
+
+public:
```

Comment:
Is making the query profilers members public necessary? As a general rule exposing implementation details can lead to many problems, especially w.r.t. member variables. There needs to be a good reason for member variables to be public.